M/MParT/build_tarballs.jl

```diff
@@ -21,28 +32,38 @@ fi
 
 cmake -DCMAKE_INSTALL_PREFIX=$prefix \
   -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+  -DCMAKE_CXX_FLAGS="-I${prefix}/include/eigen3 -fopenmp" \
```

Comment:
```suggestion
  -DCMAKE_CXX_FLAGS="-I${includedir}/eigen3 -fopenmp" \
```

Reply:
I'll make this change, but if this is how we should be doing it, should the [this documentation](https://docs.binarybuilder.org/stable/troubleshooting/#Header-files-of-the-dependencies-can't-be-found) be updated accordingly? I'd be happy to put up a quick PR, I just wasn't aware of this variable (and I'm not sure why I would be).

Reply:
Yes please, that was probably before the `${includedir}` variable was introduced.